apps/notifications/src/displayNotifications.js

```diff
@@ -0,0 +1,19 @@
+const { ConsoleTerminalProvider, Encoding, FileSystem, Terminal } = require('@rushstack/node-core-library');
+
+const pathToNotificationJson = 'common/config/notifications/notifications.json';
+const result = FileSystem.readFile(pathToNotificationJson, { encoding: Encoding.Utf8 });
```

Comment:
This is the script that is supposed to invoke Git to fetch the notifications file. The branch to pull from needs to be overrideable by CLI parameter, since for local development we will not initially be able to pull from `main` (or `master`, as rushstack is currently on) initially.